packages/template-retail-react-app/app/components/item-variant/item-attributes.jsx

```diff
@@ -80,7 +80,10 @@ const ItemAttributes = ({includeQuantity, currency, ...props}) => {
                             <FormattedNumber
                                 style="currency"
                                 currency={currency || basket?.currency || activeCurrency}
-                                value={variant.priceAdjustments[0].price}
+                                value={variant.priceAdjustments.reduce(
+                                    (acc, adj) => acc + (adj.price ?? 0),
```

Comment:
@clavery Correct me if I'm wrong but this is just a UI fix correct?

In the backend, are all price adjustments being applied to the item?

Reply:
Correct. The data/basket and checkout is all correct. But the UI here is incorrect when there is > 1 priceAdjustment as the current state is looking at the first element only.